### PR TITLE
Add dask-awkward

### PR DESCRIPTION
**Pull request by douglasdavis**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/awkward-pandas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  host:
+    - python >=3.8
+    - pip
+    - hatchling >=1.8.0
+    - hatch-vcs
+    - toml
+  run:
+    - python >=3.8
+    - awkward >=1.10.1
+    - dask
```
Comment (dopplershift) [inline recipes/awkward-pandas/meta.yaml:27]: Any particular reason this doesn't match upstream with:
```suggestion
    - dask >=2022.01.0
```
?

**Review by dopplershift — COMMENTED** [review]
Minor question. I also need @martindurant to confirm that they are willing to be listed as a maintainer for this repo.

**Comment by martindurant** [comment]
I confirm

File: recipes/awkward-pandas/meta.yaml (hunk 1 above)
Comment (douglasdavis) [reply recipes/awkward-pandas/meta.yaml]: I meant to go back and check when Dask dropped Python 3.7 add that version (happens to be `dask >=2022.02.1`). Going to add this in both places.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**